test/blocks/caas/mocks/utils.js

```diff
@@ -25,6 +25,9 @@ export function createIntersectionObserver({ el, callback /* , once = true, opti
   callback(el, { target: el });
 }
 
+const PAGE_URL = new URL(window.location.href);
+export const SLD = PAGE_URL.hostname.includes('.aem.') ? 'aem' : 'hlx';
```

Comment:
Nit: does this actually need to be exported?

Reply:
If the tests import a module that imports SLD, it is needed yes. Since this is a replication of the utils, it should be exported similar to the utils